### PR TITLE
dcat-issue-983 - comments -> definitions

### PR DESCRIPTION
**Pull request by dr-shorthair**
Responding to #983 

Copied all rdfs:comments to skos:definitions for DCAT elements

### CODE REVIEW

File: dcat/rdf/dcat.ttl [hunk 1]
```diff
@@ -1,235 +1,580 @@
...
+@prefix time: <http://www.w3.org/2006/time#> .
+@prefix vcard: <http://www.w3.org/2006/vcard/ns#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+dct:Location
+  skos:changeNote "Class added in this context in DCAT 2.0."@en ;
+  skos:changeNote "Esta clase se agregó en este contexto en DCAT 2.0."@es ;
+  skos:changeNote "Třída v tomto kontextu byla přidána ve verzi DCAT 2.0"@cs ;
+  skos:definition "A spatial region or named place."@en ;
+  skos:definition "A spatial region or named place."@es ;
```
Comment (riccardoAlbertoni) [inline dcat/rdf/dcat.ttl]:
```suggestion
  skos:definition "Una región espacial o un lugar con nombre."@es ;
```

File: dcat/rdf/dcat.ttl [hunk 2]
```diff
@@ -1,235 +1,580 @@
...
+  skos:scopeNote "En el contexto de DCAT 2.0, pro geografický střed prostorové oblasti či jiný charakteristický bod BY MĚLA být použita vlastnost dcat:centroid."@cs ;
+  skos:scopeNote "En el contexto de DCAT 2.0, pro rozsáhlé geometrie (tj. sady souřadnic určujících relevantní geografickou oblast) BY MĚLA být použita vlastnost locn:geometry [[LOCN]]."@cs ;
+  skos:scopeNote "In the context of DCAT 2.0, for a geographic bounding box delimiting a spatial area the property dcat:bbox SHOULD be used."@en ;
+  skos:scopeNote "In the context of DCAT 2.0, for an extensive geometry (i.e., a set of coordinates denoting the vertices of the relevant geographic area), the property locn:geometry [[LOCN]] SHOULD be used."@en ;
+  skos:scopeNote "In the context of DCAT 2.0, for the geographic center of a spatial area, or another characteristic point, the property dcat:centroid SHOULD be used."@en ;
+.
+dct:PeriodOfTime
+  skos:changeNote "Class added in this context in DCAT 2.0."@en ;
+  skos:changeNote "En el contexto de DCAT 2.0, esta clase se agregó en este contexto en DCAT 2.0."@es ;
```
Comment (riccardoAlbertoni) [inline dcat/rdf/dcat.ttl]:
```suggestion
  skos:changeNote "Esta clase se agregó en este contexto en DCAT 2.0."@es ;
```

File: dcat/rdf/dcat.ttl [hunk 3]
```diff
@@ -1,235 +1,580 @@
...
+  skos:definition "Temporal characteristics of the resource."@en ;
+  skos:definition "V kontextu DCAT 2.0 dcat:Dataset, časové pokrytí datové sady."@cs ;
+  skos:scopeNote "En el contexto de DCAT 2.0 dcat:Dataset, la cobertura temporal del conjunto de datos puede codificarse como una instancia de dct:PeriodOfTime, o puede indicarse usando una referencia URI (es decir, un enlace) a un recurso que describa el período o interválo de tiempo."@es ;
+  skos:scopeNote "In the context of DCAT 2.0 dcat:Dataset, the temporal coverage of a dataset may be encoded as an instance of dct:PeriodOfTime, or may be indicated using a URI reference (link) to a resource describing a time period or interval."@en ;
+  skos:scopeNote "V kontextu DCAT 2.0 dcat:Dataset, časové pokrytí datové sady může být zaznamenáno jako instance dct:PeriodOfTime, nebo může být indikováno jako IRI odkazující na zdroj popisující či časový interval."@cs ;
+.
+dct:title
+  skos:definition "A name given to the resource."@en ;
+  skos:definition "En el contexto de DCAT 2.0 dcat:Distribution, un nombre o título dado a la distribución."@en ;
```
Comment (riccardoAlbertoni) [inline dcat/rdf/dcat.ttl]:
```suggestion
  skos:definition "En el contexto de DCAT 2.0 dcat:Distribution, un nombre o título dado a la distribución."@es ;
```

File: dcat/rdf/dcat.ttl [hunk 4]
```diff
@@ -239,1389 +584,1249 @@ dcat:CatalogRecord
...
+  skos:changeNote "New class added in DCAT 2.0."@en ;
+  skos:changeNote "Nová třída přidaná ve verzi DCAT 2.0"@cs ;
+  skos:changeNote "Nueva clase agregada en DCAT 2.0."@es ;
+  skos:changeNote "Nuova classe aggiunta in DCAT 2.0."@it ;
+  skos:definition "Recurso publicado o curado por un agente único."@es ;
+  skos:definition "Resource published or curated by a single agent."@en ;
+  skos:definition "Risorsa pubblicata o curata da un singolo agente."@it ;
+  skos:definition "Zdroj publikovaný či řízený jediným činitelem"@cs ;
+  skos:editorialNote "Estado: esta clase se aceptó por el equipo de revisión de DCAT, pero los detalles de la definición y restricciones no están resueltos aún."@es ;
+  skos:editorialNote "Status: Class accepted by DCAT revision team, but details of definition and constraints not yet resolved."@en ;
```
Comment (riccardoAlbertoni) [inline dcat/rdf/dcat.ttl]: I think we have finalized the definition of dcat:Resource, so we can drop this editorial note and the related translations
```suggestion
```

File: dcat/rdf/dcat.ttl [hunk 5]
```diff
@@ -239,1389 +584,1249 @@ dcat:CatalogRecord
...
+    ] ;
+  skos:changeNote "New class added in DCAT 2.0."@en ;
+  skos:changeNote "Nová třída přidaná ve verzi DCAT 2.0"@cs ;
+  skos:changeNote "Nueva clase agregada en DCAT 2.0."@es ;
+  skos:changeNote "Nuova classe aggiunta in DCAT 2.0."@it ;
+  skos:definition "Recurso publicado o curado por un agente único."@es ;
+  skos:definition "Resource published or curated by a single agent."@en ;
+  skos:definition "Risorsa pubblicata o curata da un singolo agente."@it ;
+  skos:definition "Zdroj publikovaný či řízený jediným činitelem"@cs ;
+  skos:editorialNote "Estado: esta clase se aceptó por el equipo de revisión de DCAT, pero los detalles de la definición y restricciones no están resueltos aún."@es ;
```
Comment (riccardoAlbertoni) [inline dcat/rdf/dcat.ttl]: I think we have finalized the definition of dcat:Resource, so we can drop this editorial note and the related translations
```suggestion
```

File: dcat/rdf/dcat.ttl [hunk 6]
```diff
@@ -239,1389 +584,1249 @@ dcat:CatalogRecord
...
+  skos:editorialNote "Status: Class accepted by DCAT revision team, but details of definition and constraints not yet resolved."@en ;
+  skos:editorialNote "Stav: Třída přijata týmem pro revizi DCAT, ale detaily definice a omezení ještě nejsou vyřešeny."@cs ;
```
Comment (riccardoAlbertoni) [inline dcat/rdf/dcat.ttl]: I think we have finalized the definition of dcat:Resource, so we can drop this editorial note and the related translations
```suggestion
```

File: dcat/rdf/dcat.ttl [hunk 7]
```diff
@@ -239,1389 +584,1249 @@ dcat:CatalogRecord
...
+  skos:scopeNote "dcat:Resource is an extension point that enables the definition of any kind of catalog. Additional subclasses may be defined in a DCAT profile or application for catalogs of other kinds of resources."@en ;
+  skos:scopeNote "dcat:Resource je bod pro rozšíření umožňující definici různých druhů katalogů. Další podtřídy lze definovat v profilech DCAT či aplikacích pro katalogy zdrojů jiných druhů"@cs ;
+  skos:scopeNote "dcat:Resource è un punto di estensione che consente la definizione di qualsiasi tipo di catalogo. Sottoclassi aggiuntive possono essere definite in un profilo DCAT o in un'applicazione per cataloghi di altri tipi di risorse."@it ;
+.
+dcat:Role
+  a owl:Class ;
+  rdfs:comment "A role is the function of a resource or agent with respect to another resource, in the context of resource attribution or resource relationships."@en ;
+  rdfs:comment "Role je funkce zdroje či agenta ve vztahu k jinému zdroji, v kontextu přiřazení zdrojů či vztahů mezi zdroji."@en ;
```
Comment (riccardoAlbertoni) [inline dcat/rdf/dcat.ttl]:
```suggestion
  rdfs:comment "Role je funkce zdroje či agenta ve vztahu k jinému zdroji, v kontextu přiřazení zdrojů či vztahů mezi zdroji."@cs ;
```

File: dcat/rdf/dcat.ttl [hunk 8]
```diff
@@ -239,1389 +584,1249 @@ dcat:CatalogRecord
...
+  rdfs:label "Role"@en ;
+  rdfs:label "Ruolo"@it ;
+  rdfs:seeAlso dcat:hadRole ;
+  rdfs:subClassOf skos:Concept ;
+  skos:changeNote "New class added in DCAT 2.0."@en ;
+  skos:changeNote "Nová třída přidaná ve verzi DCAT 2.0"@cs ;
+  skos:changeNote "Nueva clase agregada en DCAT 2.0."@es ;
+  skos:changeNote "Nuova classe aggiunta in DCAT 2.0."@it ;
+  skos:definition "A role is the function of a resource or agent with respect to another resource, in the context of resource attribution or resource relationships."@en ;
+  skos:definition "Role je funkce zdroje či agenta ve vztahu k jinému zdroji, v kontextu přiřazení zdrojů či vztahů mezi zdroji."@en ;
```
Comment (riccardoAlbertoni) [inline dcat/rdf/dcat.ttl]:
```suggestion
  skos:definition "Role je funkce zdroje či agenta ve vztahu k jinému zdroji, v kontextu přiřazení zdrojů či vztahů mezi zdroji."@cs ;
```

File: dcat/rdf/dcat.ttl [hunk 9]
```diff
@@ -239,1389 +584,1249 @@ dcat:CatalogRecord
...
+  rdfs:range rdfs:Resource ;
+  owl:propertyChainAxiom (
+      dcat:accessService
+      dcat:endpointURL
+    ) ;
+  skos:definition """A URL of a resource that gives access to a distribution of the dataset. E.g. landing page, feed, SPARQL endpoint.
+          Use for all cases except a simple download link, in which case downloadURL is preferred."""@en ;
+  skos:definition """Ceci peut être tout type d'URL qui donne accès à une distribution du jeu de données. Par exemple, un lien à une page HTML contenant un lien au jeu de données,
+					un Flux RSS, un point d'accès SPARQL. Utilisez le lorsque votre catalogue ne contient pas d'information sur quoi il est ou quand ce n'est pas téléchargeable."""@fr ;
+  skos:definiti
```
Comment (riccardoAlbertoni) [inline dcat/rdf/dcat.ttl]: To stay coherent with the rest of the dcat,  I suspect we should add rdfs:comments for these skos:definition.

Comment (dr-shorthair) [reply dcat/rdf/dcat.ttl]: Not necessarily. We should retain `rdfs:comment` where it was used in DCAT 2014. But if we prefer `skos:definition` moving forward, then we don't necessarily need to duplicate into `rdfs:comment` for new elements.

Comment (pwin) [reply dcat/rdf/dcat.ttl]: I agree with @dr-shorthair about not being bound by previous practices ... not only skos:definition, but also other skos annotations could be very helpful fine-grained documentation 
[https://www.w3.org/TR/skos-primer/#secdocumentation](https://www.w3.org/TR/skos-primer/#secdocumentation)

**Review by davebrowning — APPROVED** [review]

**Review by riccardoAlbertoni — APPROVED** [review]

**Pull request merged**